app-shells/bash_completion/patches/bash_completion-2.11.patchset

```diff
@@ -0,0 +1,102 @@
+From c83f6af3e75d67aa7789c5656f872c4b3aaf49cc Mon Sep 17 00:00:00 2001
+From: JakobDev <jakobdev@gmx.de>
```

Comment:
Would be nice if the credits were kept as they were (quick look doesn't show any real diffs):
https://github.com/haikuports/haikuports/blob/f85fb2d8ddbc31ce149a3395a48ff1cca61c05c9/app-shells/bash_completion/patches/bash_completion-2.7.patchset#L2